vllm/worker/model_runner.py

```diff
@@ -879,7 +879,7 @@ def capture(
         # Run the model once without capturing the graph.
         # This is to make sure that the captured graph does not include the
         # kernel launches for initial benchmarking (e.g., Triton autotune).
-        with _maybe_cupy_nccl():
+        with _maybe_pynccl():
```

Comment:
QQ: Can we replace PyTorch NCCL with our PyNCCL (except for the initialization part)?

Reply:
I think we can, but not sure how we can confirm it. Let's make `pynccl` as default, and see the community usage report. Then we can replace PyTorch NCCL in our next release.